node/lib/util/commit.js

```diff
@@ -301,23 +362,26 @@ exports.commit = co.wrap(function *(metaRepo, all, metaStatus, message) {
     const subCommitters = Object.keys(submodules).map(commitSubmodule);
```

Comment:
There are libraries like http://caolan.github.io/async/docs.html#each which maybe make this more explicit?   I don't use enough node to really know what's idiomatic, so maybe I'm wrong to find this ugly.